src/sass/feature/_base.scss

```diff
@@ -4,50 +4,58 @@
 @use 'core' as *;
 
 .#{$prefix}-feature {
+  display: flex;
+  flex-direction: column;
+
+  &--horizontal {
+    flex-direction: row;
+
+    .#{$prefix}-feature__body :first-child {
```

Comment:
So with this one, given the weird issues I was experiencing with the lobotomized owl, I needed to guarantee that on the horizontal layout, the first element within `__body` wouldn't have a `margin-top`. When I erase the the space, it targets differently than if I keep the space.

This is what the targeting description is within VS Code with the space:
<img width="473" alt="Screen Shot 2020-03-05 at 10 48 25 AM" src="https://user-images.githubusercontent.com/15944675/75998793-5c6b5280-5ecf-11ea-95f5-b27531ba03f3.png">

This is the description without the space:
<img width="553" alt="Screen Shot 2020-03-05 at 10 48 35 AM" src="https://user-images.githubusercontent.com/15944675/75998864-75740380-5ecf-11ea-9f3e-c9ce73175305.png">

Reply:
Cool, I'm checking it out now. That's weird. I've never ran into a situation like that. 🤔 Seems like then in this case you're just targeting the pseudo element `:first-child` in general then? That's wild, I've never seen that before.

Reply:
Oh, I think I see why this is happening. The `> * + *` selector should take care of not applying top margin to the first element inside of `__body`, but in these examples the first element is a `p` tag for the first `__detail` element. There's a more specific rule up the cascade, if you disable that rule you can see it's the default user agent style sheet and that is somehow more specific than the lobotomized owl here. 

![image](https://user-images.githubusercontent.com/5590034/76003021-7f990080-5ed5-11ea-912d-7019ea9e8356.png)

If that first `__detail` element were a generic `div` or `span` it's not a problem, but it should be tolerant enough for folks to use different elements to mark it up if needed.

The thing that threw me off was the general child select combo of the `__body :first-child`. To make it a little clearer, it would probably make sense to use the direct child and universal selector there. 

So something like:

```sass
.#{$prefix}-feature__body > *:first-child {
  margin-top: 0;
}
```

This will also make it so _only_ the first direct child gets the top margin set to 0. I could imagine a scenario where there might be elements nested inside of the a `rvt-feature__detail` element that you would want to be able to control the top margin on. the way the selector is written now, it would set the `margin-top: 0` for _any_ `:first-child` inside of _any_ `__detail` element, no matter how deeply-nested.

One more thought here. I think it would make sense to set the `__detail` element to display block by default. If I change the markup where you have "Publish date" to a `<time>` element, which would make sense in a lot of editorial contexts, the `margin-top` rule in the lobotomized owl doesn't get applied because `time` is an inline, not block element to which margins won't get applied.

Reply:
Ahhhh. That makes so much sense! Thanks for the assist, I'll get on updating that. 🙌